### DISCUSSION TITLE
Null synchronization cloud - pc

### DISCUSSION THREAD

**Discussion by mediosapoyoconservacion** [community/community#53676] [Mobile]
### Select Topic Area

Question

### Body

After making changes to the mobile and uploading it to the cloud. These modifications appear in the cloud, but when I want to synchronize with the PC, these modifications are not detected and it asks me to upload the file that I have on the PC, when I really should download the one from the cloud.
Any explanation or solution to this issue?

**Comment by taki-on** [comment]
Hello! This forum is about features and bugs in the GitHub Mobile apps. It seems your question is not relevant to this forum. You might want to find the right place to ask your question. Thank you!
